doc/source/reference/functions/string_slice.rst

```diff
@@ -0,0 +1,211 @@
+.. -*- rst -*-
+
+.. highlightlang:: none
+
+.. groonga-command
+.. database: functions_string_slice
+
+``string_slice``
+================
+
+.. versionadded:: 11.0.2
```

Comment:
Could you change to 11.0.3 from 11.0.2.
Because Groonga 11.0.2 has already released.

Reply:
I fixed it.